go/chat/helper.go

```diff
@@ -601,7 +648,10 @@ func (n *newConversationHelper) create(ctx context.Context) (res chat1.Conversat
 		case chat1.ConversationMembersType_TEAM:
 			n.topicName = &DefaultTeamTopic
 		}
+	} else if n.membersType == chat1.ConversationMembersType_IMPTEAM {
```

Comment:
This isn't quite right, we shouldn't block this for `TopicType_DEV`, I would just remove this check.

Reply:
removed